.github/workflows/build-test-pr.yml

```diff
@@ -199,6 +209,61 @@ jobs:
         with:
           folder: ${{ matrix.folder }}
 
+  test-project-folders:
+    runs-on: ubuntu-18.04
+    needs: [ build-python ]
+    strategy:
+      matrix:
+        folder: [ hi-ml-histopathology ]
+        packageName: [ '*.whl' ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Set up Python ${{ env.pythonVersion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: PIP upgrade
+        run: |
+          cd hi-ml-azure
+          make pip_upgrade
+
+      - name: Download hi-ml-azure
+        id: download_himlazure
+        uses: ./.github/actions/download_package_artifacts
+        with:
+          folder: hi-ml-azure
+          package_type: ${{ matrix.packageName }}
+
+      - name: Download hi-ml
+        id: download_himl
+        uses: ./.github/actions/download_package_artifacts
+        with:
+          folder: hi-ml
+          package_type: ${{ matrix.packageName }}
+
+      - name: Install dependencies
+        run: |
+          cd hi-ml-azure
+          pip install ${{ steps.download_himlazure.outputs.package_filename }} 
+          cd ../hi-ml
+          pip install ${{ steps.download_himl.outputs.package_filename }}
+
+      - name: Run tests
+        run: |
+          cd ${{ matrix.folder }}
+          make pip_local
+          make pytest_and_coverage
```

Comment:
The documentation developers.md talks about "make test" this does not work as a make command. Also this other options are not mentioned